src/main/java/de/tum/in/www1/artemis/domain/BuildJob.java

```diff
@@ -58,15 +58,15 @@ public class BuildJob extends DomainObject {
 
     @Enumerated(EnumType.STRING)
     @Column(name = "build_job_result")
```

Comment:
Should we rename this column in the database? If yes, I'd do this before the 7.0.0 release so that the new initial schema already contains the name. [Rename can be done in a followup]

Reply:
Yes sounds good. I'll do it in a followup!